docs/dictionary/function/processor.lcdoc

```diff
@@ -23,17 +23,15 @@ if the processor is "ARM" then ...
 
 Returns (enum):
 
--   x86: Windows systems (any architecture) and Mac OS X and Linux
-    systems with a 32 bit processor.
--   x86_64: Mac and Linux systems running a 64 bit processor
--   i386: iOS Simulator.
--   ARM: iOS and Android Devices and the Android simulator
-
+-   x86: 32 or 64 bit Windows systems and 32 bit macOS or Linux systems.
```

Comment:
I never realized this about 'the processor'... I just assumed it was the running machine. Is there an actual situation where this would be useful (aside from the trivial 'what processor was the engine compiled on?')... I can't come up with one. Seems *so* much more useful to have 'the processor' return info about the runtime system.